Source/WebCore/inspector/InspectorCanvas.cpp

```diff
@@ -107,18 +107,18 @@ Ref<InspectorCanvas> InspectorCanvas::create(CanvasRenderingContext& context)
 
 InspectorCanvas::InspectorCanvas(CanvasRenderingContext& context)
     : m_identifier("canvas:" + IdentifiersFactory::createIdentifier())
-    , m_context(context)
+    , m_context(&context)
 {
 }
 
 HTMLCanvasElement* InspectorCanvas::canvasElement() const
 {
-    return dynamicDowncast<HTMLCanvasElement>(m_context.canvasBase());
+    return dynamicDowncast<HTMLCanvasElement>(m_context->canvasBase());
```

Comment:
This needs a `RefPtr<> protectedContext` to keep it alive during the method call, and possibly a `nullptr` check to prevent a crash.

Same for most uses of `m_context` in this class.